sections/remotes.tex

```diff
@@ -161,6 +163,7 @@ \subsection{Forking Workflow - Clone}
   HEAD branch: master
 \end{lstlisting}
 
+  Make sure to change into the newly cloned directory with \lstinline{cd agile-ipsum}.
```

Comment:
I'd prefer to have this after the `git clone` in the code block, to have a smooth set of commands to execute instead of having this "subtle" note after the fact ...

Reply:
put `cd ~` and `cd agile-ipsum` inside the code block